painter/src/painter.rs

```diff
@@ -339,13 +339,22 @@ impl Painter {
     style: &TextStyle,
     foreground: Brush,
     bounds: Option<Size>,
+    overflow: Overflow,
```

Comment:
overflow should not be a config for painter, just for text widget.

Reply:
The api of text drawing needs to be modified. Currently, text drawing will rely on Typography cache, which should not be